### PR TITLE
79 Suggested Questions

### PR DESCRIPTION
**Pull request by rurumeister**
Screenshot 2024-12-11 at 3 52 49 PM

Add Suggested Questions 

- Create new Information Icon
- Create Query Suggestion Component
- Added Suggestion Component to Chatbar

### CODE REVIEW

**Review by chunxtan — APPROVED** [review]
spelling + minor styling suggestion to add 
`.chatBar textarea {
    padding-top: 0.3rem;
    padding-bottom: 0.3rem;
}` to `chat-bar.module.css` file.

This is to align the middle of the (i) icon to the text.
image

else LGTM

File: frontend/src/components/query-suggestions/query-suggestions.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { InfoIcon } from "../../assets/icons/info-icon";
+import classes from "./query-suggestions.module.css";
+
+interface QuerySuggestionsProps {
+  setUserInput: (input: string) => void;
+}
+
+const QuerySuggestions = ({ setUserInput }: QuerySuggestionsProps) => {
+  const suggestions = [
+    "Can you summarize the eligibility requirements and benefits for <insert scheme name here>?",
```
[the remark this answers is not in the document]
Comment (rurumeister) [reply frontend/src/components/query-suggestions/query-suggestions.tsx:16]: fixed!
